tests/chimp-config.js

```diff
@@ -19,7 +19,7 @@ module.exports = {
 // 	showXolvioMessages: true,
 
 // 	// - - - - CUCUMBER - - - -
-	path: 'tests/steps',
+	path: 'tests',
```

Comment:
@rodrigok Is this good? Or you think that i should create a new folder and put "end-to-end-api" and "end-to-end-ui" folders in there and set the path to that folder?